src/SWP/Bundle/BridgeBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+    # move it into an appropriate place (CoreBundle?)
+    swp.serializer:
+        class: SWP\Bridge\JMSSerializerBundle\JMSSerializer
```

Comment:
I created it because I wanted to decouple it from the project. We need serializer for the data transformer (payload => Package). This  transformer is placed inside bridge component, I didnt want to make a dependency on JMS serializer so I created our Serializer Interface which doesnt require JMS serializer at all. JMS serializer implements our serializer interface which makes it possible to use it in our project without hard dependency. Data transformer can work with various serializers with this approach and is not coupled with the JMS bundle itself.